ReactWindows/ReactNative.Shared/UIManager/LayoutShadowNode.cs

```diff
@@ -10,7 +10,11 @@ namespace ReactNative.UIManager
     /// </summary>
     public class LayoutShadowNode : ReactShadowNode
     {
+#if WINDOWS_UWP
         private const float Undefined = CSSConstants.Undefined;
+#else
+        private const float Undefined = 0.0f;
```

Comment:
Are you sure this is the correct undefined value?  What is the problem with using CSSConstants.Undefined?